modules/using-the-build-time-network-policy-generator.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
```

Comment:
Content wise I cannot add much to this command (probably someone from Maple team can fill some gaps), but one thing to note is that this is tech preview, this should probably be called out here?

Reply:
It is a tech preview, but looks like although it is [noted elsewhere](https://docs.openshift.com/acs/4.1/operating/manage-network-policies.html#using-build-time-network-policy-generator) in the docs as TP, but the TP snippet/language is in the assembly there, not the module. I would add the TP snippet to the module here too. It might have gotten deleted from the module since it would make it show up twice on [this page](https://docs.openshift.com/acs/4.1/operating/manage-network-policies.html#using-build-time-network-policy-generator), just a guess.

Reply:
added tech preview snippet